config/local/gateway-service.yml

```diff
@@ -7,7 +7,7 @@ apiml:
         discoveryServiceUrls: https://localhost:10011/eureka/
     security:
         auth:
-            provider: dummy
+            provider: zosmf
```

Comment:
because just reading this should tell it's a mock

Reply:
I get it but it may bring some issues

ZOSMF("zosmf","zosmfAuthenticationProvider"),

Here I thing there is no reasonable way to change it.